release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -539,6 +539,13 @@ Benefits of this feature include faster data processing, reduced CPU workloads,
 
 For more information, see xref:../networking/hardware_networks/configuring-hardware-offloading.adoc#about-hardware-offloading_hardware-offloading[Configuring hardware offloading].
 
+[id="ocp-4-10-networking-external-dns-operator"]
+==== Creating DNS records by using Red Hat External DNS Operator (Technology Preview)
```

Comment:
by using Red Hat External DNS Operator -> by using the Red Hat External DNS Operator

Reply:
Addressed